beacon-chain/sync/initial-sync/round_robin.go

```diff
@@ -314,14 +314,24 @@ func (s *Service) processBatchedBlocks(ctx context.Context, genesis time.Time,
 			errParentDoesNotExist, first.Block().ParentRoot(), first.Block().Slot())
 	}
 	s.logBatchSyncStatus(genesis, first, len(bwb))
+	blobCount := 0
 	for _, bb := range bwb {
 		if len(bb.Blobs) == 0 {
 			continue
 		}
 		if err := s.cfg.DB.SaveBlobSidecar(ctx, bb.Blobs); err != nil {
 			return errors.Wrapf(err, "failed to save blobs for block %#x", bb.Block.Root())
 		}
+		blobCount += len(bb.Blobs)
 	}
+	if len(bwb) > 0 {
```

Comment:
I just noticed we index into `bwb[0]` on line 311, which is a potential panic. Can we change this to be an early return right after the call to `validUnprocessed` to protect from that panic? Also just want to double check that you want this log to happen when there are zero blobs in the list. Otherwise we should change this to `if blobCount > 0`

Reply:
Done and done [5b1971f](https://github.com/prysmaticlabs/prysm/pull/12883/commits/5b1971f8517aef32333cb012148ff0eb6f47c964)